### PR TITLE
Add pedal dimensions to panel

### PR DESCRIPTION
**Pull request by tandrewnichols**
There've been some times messing with pedals where I couldn't find dimensions for a pedal, but I knew it was "about the same as pedal X" but there's no easy way to see the dimensions for pedals. Also, if you add a pedal and then accidentally delete it or want to add a new one with the same dimensions there's no way to know what they are. I just added `(W x H)` below the name in the side panel. Also, I added a section to the readme about running locally. Since you're not using a server, you have to run Chrome with CORS disabled or else none of the resources serve.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,3 +17,16 @@
 * Add the dimensions of the pedals you add to `/public/data/pedals.json`
 * Input dimensions in inches, with decimals (rounding to nearest hundredth)
 * Dimensions recorded should include jacks and switches (if possible, most placed don't specify wether or not measurements do or do not)
+
+# Running Locally
+
+1. `npm install`
+2. `bower install`
+3. `gulp`
+4. Start Chrome with web security (CORS) disabled via:
```
Comment (tehtrav) [inline README.md]: I'd rather not instruct users to disable security features

File: app/scripts/scripts.js [hunk 2]
```diff
@@ -529,8 +529,13 @@ $('body').on('click', '.item', function(e){
 	var pedal       = $(this);
 	var id   		= $(this).attr("id");
 	var pedalName   = $(this).attr('title');
+  var width       = $(this).attr('data-width');
+  var height      = $(this).attr('data-height');
 	var markup      = '<div class="panel" data-id="#'+id+'">\
-		<div class="panel__name">'+pedalName+'</div>\
+    <div class="panel__name">'
```
Comment (tehtrav) [inline app/scripts/scripts.js]: Good idea, I can def see this being useful!

File: README.md (hunk 1 above)
Comment (tandrewnichols) [reply README.md]: I'm curious how you run this locally then. There's no web server, so you have to just open files in a browser, but if you do that, all the pedal and pedalboard images are blocked by CORS. The only way I could run it was a new instance of Chrome with CORS disabled.

Comment (tandrewnichols) [reply README.md]: Actually, [gulp-webserver](https://www.npmjs.com/package/gulp-webserver) looks a little easier to use (or at least, there's better documentation).

Comment (tehtrav) [reply README.md]: Sure! I have no experience with it but it looks fine to me

**Comment by tandrewnichols** [comment]
@tehtrav The gulp servers I mentioned would require changing how resources are structured (`index.html` needs to be in `public` and script/style urls need to then have `public/` removed), so I added `http-server` instead, which is a simple static file server for node. Really easy to use. Start it with `npm run serve` after running `gulp`.

**Comment by tandrewnichols** [comment]
@tehtrav Are you waiting for some time to test this on your end?

**Comment by tehtrav** [comment]
Yup! Sorry, haven't had a chance to run this yet. Will try to do that this weekend.

**Comment by tandrewnichols** [comment]
No worries, just checking in.

**Comment by tehtrav** [comment]
I'm sure its fine... merged!

**Pull request merged**